scripts/checkpoint_averaging/checkpoint_averaging.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Builds a .nemo file with average weights over multiple .ckpt files (assumes .ckpt files in same folder as .nemo file).
+
+Usage example for building *-averaged.nemo for a given .nemo file:
+
+NeMo/scripts/checkpoint_averaging/checkpoint_averaging.py my_model.nemo
+
+Usage example for building *-averaged.nemo files for all results in sub-directories under current path:
+
+find . -name '*.nemo' | grep -v -- "-averaged.nemo" | xargs NeMo/scripts/checkpoint_averaging/checkpoint_averaging.py
+"""
+
+import glob
+import os
+import sys
+
+import torch
+
+from nemo.core import ModelPT
+from nemo.utils import logging, model_utils
+
+
+def main():
+    device = torch.device("cpu")
+
+    model_fname_list = sys.argv[1:]
```

Comment:
Why not just use argparse for future changes?

Reply:
I typically use it, but found it an overkill here.
I can use it here as well if you believe it to be useful.

Reply:
For the sake of uniformity I switched to argsparse.